store/copr/mpp.go

```diff
@@ -256,13 +278,21 @@ func (m *mppIterator) establishMPPConns(bo *tikv.Backoffer, req *kv.MPPDispatchR
 		return
 	}
 
-	// TODO: cancel the whole process when some error happens
 	for {
 		err := m.handleMPPStreamResponse(bo, resp, req)
 		if err != nil {
+			m.cancelMppTask(bo, req, taskMeta)
```

Comment:
I think it only cancel the mpp task of one store, since TiDB is kind of coordinator, is it possible to kill the mpp task in all stores if one store meet error?

Reply:
updated, sending cancel cmd to all stores.